### DISCUSSION TITLE
Lock map

### DISCUSSION THREAD

**Discussion by davemenkehorst** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#355] [Q&A] [answered]
Great great component Piotr.

I use the map extractor every day on my home screen lovelace. If i scroll over on my dashboard and scroll 'over' the map, it accidentally zoom out on the map. 
There is a 'lock' button on the top right in the map. Can i default it to the locked position?
TY

**Comment by PiotrMachowski** [comment, marked as answer]
This is a feature of a map card (I have transferred the discussion).

Yes, it is even in editor:

![image](https://user-images.githubusercontent.com/6118709/153378761-d916f411-5040-4273-8126-46849983359b.png)

or in yaml

```yaml
map_locked: true
```

**Reply by davemenkehorst** [reply]
Then i looked over it. Sorry
